src/main/java/net/snowflake/ingest/streaming/internal/StreamingIngestResponseCode.java

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright (c) 2021 Snowflake Computing Inc. All rights reserved.
+ */
+
+package net.snowflake.ingest.streaming.internal;
+
+public enum StreamingIngestResponseCode {
+  SUCCESS(0, "Success"),
+  ERR_INVALID_REQUEST(1, "The given request has an invalid format"),
+  ERR_CHANNEL_NAME_EXISTS(2, "The channel name already exists"),
+  ERR_INVALID_WRITE_MODE(3, "The supplied write mode is not valid"),
+  ERR_TABLE_DOES_NOT_EXIST_NOT_AUTHORIZED(
+      4, "The supplied table does not exist or is not authorized"),
+  ERR_CHANNEL_LIMIT_EXCEEDED_FOR_TABLE(5, "Channel limit exceeded for the given table"),
+  ERR_TABLE_OR_COLUMN_TYPE_NOT_SUPPORTED(
+      6,
+      "Snowpipe Streaming is not supported on the type of table resolved, or the table contains"
+          + " columns that are either AUTOINCREMENT, IDENTITY or has default value"),
+  ERR_ENQUEUE_TABLE_CHUNK_QUEUE_FULL(
+      7,
+      "Cannot enqueue chunks at this time due to a high number of chunks pending commit to the"
+          + " table"),
```

Comment:
We don't document chunks right? Maybe something like `The table has exceeded its limit of outstanding registration requests, please wait before calling the insertRow or insertRows API`.